src/sentry/demo/urls.py

```diff
@@ -1,9 +1,24 @@
 from django.conf.urls import url
 
+from sentry.web.frontend.react_page import ReactPageView
 from sentry.conf.urls import urlpatterns as conf_urlpatterns
 from .demo_start import DemoStartView
 
+# add the demo start route at the head of the urls
 urlpatterns = [
     url(r"^demo/start/$", DemoStartView.as_view(), name="sentry-demo-start"),
 ]
+# add the rest of our rules
 urlpatterns += conf_urlpatterns
+
+# pop off the catch all
+catch_all = urlpatterns.pop()
+# add our org catch all before the last catch all
+urlpatterns += [
```

Comment:
Do we also have this for `/settings/<org_slug>`?

Reply:
@EvanPurkhiser good call!